### DISCUSSION TITLE
深層のダンジョン内の店に夢がほしい

### DISCUSSION THREAD

**Discussion by RL-Nadir** [hengband/hengband#311] [Ideas]
少なくとも、2.xx代において、ブラックマーケット以外の店に並ぶアイテムは、ベースアイテムに制限がかかっていたとの認識で、3.0.0になっても特に変わっていないものと思われます。
地上の店はこれでいいとして、ダンジョン内の店に少し夢をもたせてほしいと考え、提案します。
具体的には、以下のような感じで、金で制圧できるvaultみたいになればと思っています。
・ダンジョン内の店の陳列品は、従来の店に加えて、ベースアイテム含めて生成階層同等（以上）が並ぶようにする
・アイテム生成レベルも、生成階層を基準として、少し幅をもたせ、たまにブーストもあり得るようにする
・深層限定のレア店舗（DSM専門店、禁断の魔導書専門店、殺意のペット専門店とか）
　ちなみに、DSM専門店はToME1-2で実例があって、PDSMが数十万で普通に並ぶ狂気の店でした。

**Comment by deskull-m** [comment]
いいですね、個人的に馬鹿馬鹿の試験実装に盛り込んだ上で変愚に逆輸入したいと思います。
https://github.com/sikabane-works/bakabakaband/issues/9

**Reply by RL-Nadir** [reply]
楽しみにしてます！
変な店ばかり思いつきますね。
・やばい薬ばかり売ってる　シグマ・エルドリッジ社
・あぶない水着とパンティとムチを売る　おとなのおみせ
・死体専門店
・お人形屋さん
（・県木のコンビニ）

**Reply by deskull-m** [reply]
＞（・県木のコンビニ）
ああ＾～
